tools/wptrunner/wptrunner/executors/executorcontentshell.py

```diff
@@ -91,7 +91,9 @@ def _read_text_block(self, deadline=None):
         while True:
             line = _read_line(self.stdout_queue, deadline, "utf-8", "replace", False)
```

Comment:
'False' here means raise_crash is False and we will never see Crash results for content shell?